addons/website_slides/models/slide_channel.py

```diff
@@ -96,23 +106,49 @@ def _recompute_completion(self):
         completed_records = self.env['slide.channel.partner']
         uncompleted_records = self.env['slide.channel.partner']
         for record in self:
+            if record.member_status == 'invited':
+                record.completed_slides_count = 0
+                record.completion = 0
+                continue
             record.completed_slides_count = mapped_data.get(record.channel_id.id, dict()).get(record.partner_id.id, 0)
-            record.completion = 100.0 if record.completed else round(100.0 * record.completed_slides_count / (record.channel_id.total_slides or 1))
+            record.completion = 100 if record.member_status == 'completed' else round(100.0 * record.completed_slides_count / (record.channel_id.total_slides or 1))
```

Comment:
This is weird. We keep a 100 completion yet we could have our karma removed below and the course set as uncompleted... Right? We should probably recompute all the time? Right now, if I go to the master runbot and complete a course, then set a slide as uncompleted, I loose karma and the completion goes down. However, If I go to the backend, I am still at 100 (it is not a computed field)... Weird behavior